lib/collection.js

```diff
@@ -267,24 +267,24 @@ const DEPRECATED_FIND_OPTIONS = ['maxScan', 'fields', 'snapshot'];
  * @param {object} [options.projection] The fields to return in the query. Object of fields to include or exclude (not both), {'a':1}
  * @param {object} [options.fields] **Deprecated** Use `options.projection` instead
  * @param {number} [options.skip=0] Set to skip N documents ahead in your query (useful for pagination).
- * @param {Object} [options.hint] Tell the query to use specific indexes in the query. Object of indexes to use, {'_id':1}
- * @param {boolean} [options.explain=false] Explain the query instead of returning the data.
```

Comment:
Do these indeed not have defaults? Is the documentation referencing a server default here?

Reply:
`explain` isn't defined as an option in the CRUD spec and is not mentioned in the [server docs](https://docs.mongodb.com/v3.4/reference/method/db.collection.find/).

Reply:
I guess my question is more: Do we actually set these values to a default value in the driver somewhere? If we do, and we are removing the info here, we are inconsistent. If we do set the values, and should not, we should not only remove the documentation but also the documentation

Reply:
I don't see anywhere that we set a default for `explain` (or any of the below options for which I've removed the default).

Reply:
:+1: